.github/workflows/node.js.yml

```diff
@@ -53,5 +53,16 @@ jobs:
       - name: Build application
         run: npm run build
 
+      - name: Confirm the build is up-to-date
+        run: |
+          REPOSTATUS=`git status --porcelain 2>/dev/null`;
+          if [[ -n "$REPOSTATUS" ]]; then
+            echo "Repo is not clean. Did you forget to commit changes to the /build dir?";
+            echo "$REPOSTATUS";
+            exit 1;
+          else
+            echo The repo file system is clean.;
```

Comment:
While true, it isn't indicative of what was actually being checked - that built files are present or not. Right now, those unfamiliar may not be able to join the dots between whether the file system is clean (is that good or bad?) and the real potential problem.

Reply:
I've added a clarification message regarding this.